src/main/kotlin/com/healthmetrix/labres/order/RegisterOrderUseCase.kt

```diff
@@ -47,18 +49,18 @@ class RegisterOrderUseCase(
             )
         }
 
-        if (existing.alreadyHasResult()) {
+        if (newestExisting.alreadyHasResult()) {
```

Comment:
Should some of the checks here and afterwards check across all orders?  For example, if an older order has a result already, should that become an error/log?

Reply:
Based on the other code, it can never happen in theory that another of the duplicate has a result while the newest one is still in progress. For me it's good enough for now leaving it like this.

The notificationUrls check I would only want to do on the newest one, as I want to treat the others as read only copies if at all.